index.js

```diff
@@ -8,10 +8,6 @@ exports.register = function (server, options, next) {
   server.plugins.icecreambar =  rollbar;
   server.plugins.icecreambar['default'] = rollbar;
 
-  server.on('request-error', function internalError (request, error) {
```

Comment:
previously `onPreRequest` didn't report `500` errors and `request-error` did; but turns out `request-error` isn't called if you have any `onPreRequest` hooks that change the reply to a non-error -- e.g. like we are doing in backboard to suppress users from seeing analytics failures.

So, I updated the `onPreRequest` in this project to report 500s now.